manim/default.cfg

```diff
@@ -127,6 +127,13 @@ save_last_frame = False
 save_pngs = False
 save_as_gif = False
 
+[custom_folders]
+media_dir = /home/jh/Downloads/temp_movie_files
+video_dir = /home/jh/Downloads/temp_movie_files
+tex_dir   = /home/jh/Downloads/temp_movie_files
+text_dir  = /home/jh/Downloads/temp_movie_files
+output_file = /home/jh/Downloads/XXX
```

Comment:
Issue: these will only work on Linux.

Issue: I don't think `Downloads` should be the output folder for manim videos. Because manim videos are not downloaded.

Issue: please remove `jh`.

Nitpick: `temp_movie_files` is redundant as everything related to manim is related to movies. I'd suggest `temp_files` or just `temp`, or even `auxiliary_files` or `aux`.

Suggestion: add another option and interpolate with the others, for example:
```
[custom_folders]
custom_root = /home/
video_dir = %(custom_root)s/aux
tex_dir = %(custom_root)s/aux
text_dir = %(custom_root)s/aux
```

For details on interpolation see [here](https://docs.python.org/3/library/configparser.html#interpolation-of-values).

Reply:
Not solved yet.
I will post my idea in the pull request thread